api/resources_portal/test/views/test_material_request.py

```diff
@@ -133,17 +139,33 @@ def test_put_request_from_sharer_updates_a_material_request(self):
     def test_put_request_from_requester_updates_a_material_request(self):
         self.client.force_authenticate(user=self.request.requester)
 
-        irb_attachment = AttachmentFactory()
+        requester_org = OrganizationFactory()
+        requester_org.members.add(self.request.requester)
+        requester_org.save()
+
+        irb_attachment = AttachmentFactory(owned_by_org=requester_org)
 
         self.material_request_data["irb_attachment"] = irb_attachment.id
-        self.material_request_data["requester_signed_mta_attachment"] = AttachmentFactory().id
+        self.material_request_data["requester_signed_mta_attachment"] = AttachmentFactory(
+            owned_by_org=requester_org
```

Comment:
I think we want to be testing `owned_by_user` in this method rather than `owned_by_org`